source/classes.tex

```diff
@@ -2078,8 +2079,13 @@
 the selected destructor may be deleted\iref{dcl.fct.def.delete}.
 
 \pnum
-\indextext{restriction!destructor}%
+\indextext{constructor!address of}%
```

Comment:
destructor

Reply:
Fixed

Reply:
Thanks -- could you rebase please? Something weird seems to have happened to this branch.